gen.sh

```diff
@@ -9,7 +9,6 @@ has() {
 cd "${gitd}"
 if [ ! -f ./discordemojimap/mapping.go ]; then if has git; then git submodule update --init; else echo "FAIL, git missing, please find your own way to download the submods"; exit 1;fi;fi
 # git isn't ""REALLY"" needed, you just have to trick the script into finding the submods
-cp ./discordemojimap/mapping.go ./
```

Comment:
well if the sub module doesn't get cloned, the line above is mostly useless...  
Do you plan to make edits to not require cloning blobmoji?  
If not then change line 10 to
```sh
if [ ! -d ./blobmoji/png ]; then if has git; then git submodule update --init; else echo "FAIL, git missing, please find your own way to download the submods"; exit 1;fi;fi
```